x/poe/types/genesis.go

```diff
@@ -81,6 +81,13 @@ func DefaultGenesisState() *GenesisState {
 					},
 					DisputeCost: sdk.NewCoin(DefaultBondDenom, sdk.NewInt(1_000_000)),
 				},
+				MixerContractConfig: &MixerContractConfig{
+					Sigmoid: MixerContractConfig_Sigmoid{
+						MaxRewards: 1000,
+						P:          sdk.MustNewDecFromStr("0.62"),
```

Comment:
I picked new values from my model. Let's try with these and see how simulations work.